src/test/regress/expected/multi_extension.out

```diff
@@ -7,6 +7,10 @@
 -- not done yet.
 --
 -- Upgrade tests verify the objects are added in citus_finish_pg_upgrade()
+--
```

Comment:
ow, second output file for multi extension test :( 

I think I'd be in favor of changing `print_extension_changes` such that it filters out any_value such that we don't have it here at all, and avoid the 2 output files

Reply:
Okay let me try that one